### DISCUSSION TITLE
Add "Full Width" as a user configurable option

### DISCUSSION THREAD

**Discussion by ssenior45** [outline/outline#5413] [Ideas]
Unless I am mistaken as of now the "Full width" option is on a per-page basis.

Personally, I want every page to be full width by default and have to remember to check this option on each page I create.

It would be nice if this boolean option could be in a user profile and the default value is taken from there. It can still be overriden on a per page basis.

**Comment by JiTmun** [comment]
Could be this be already a global settings we could set for the all instance ?
Any configuration files we could change ?
